components/homepage/HeroHeading.tsx

```diff
@@ -1,14 +1,14 @@
+import Heading1 from '@/components/shared/Heading1'
 import { TEXT } from '@/localization/texts_en'
 
 const HeroHeading = () => {
   return (
-    <h1 data-testid="hero-heading" className="text-center text-5xl font-bold sm:text-6xl lg:text-7xl">
```

Comment:
Comment for debugging. Here is some issue found later after merging.

Reply:
Fixed with: https://github.com/krsiakdaniel/portfolio-website-krsiak-cz/pull/172/files